admin/app/notify_client/models.py

```diff
@@ -117,6 +118,10 @@ def is_authenticated(self):
     def permissions(self):
         return self._permissions
 
+    @property
+    def time_zone(self):
+        return self._time_zone if self._time_zone else current_app.config['DEFAULT_TIMEZONE']
```

Comment:
This bit needs to behave a bit more defensively. In our feature env my user seems to have this field returning 'false' which makes any page with a date crash

we may need to make sure that this parses as a correct time zone here and fall back to default otherwise

![image](https://user-images.githubusercontent.com/212986/98899735-502aa100-2504-11eb-9570-5a988e2bd5c0.png)